cmd/gen-go-sample/main.go

```diff
@@ -353,7 +353,13 @@ func (g *generator) genSample(ifaceName string, methConf GAPICMethod, regTag str
 			}
 			buf.WriteByte('\n')
 		}
-		prependLines(&buf, "// ")
+		prependLines(&buf, "// ", false)
+
+		if len(files) > 0 {
```

Comment:
I'd like to hear feedback from other Go experts on preferring `:=` when there's only one file, and using this `var` structure when there's more than one, just for idiomaticity.

(That said, this could be an incremental change in a subsequent PR. Or possibly not worth doing.)

Reply:
Let's use `ioutil.ReadFile` and not even have to think about this ;-)